lib/tictactoe.rb

```diff
@@ -1,9 +1,46 @@
-class TicTacToe 
-  def renderBoard 
-    Array.new(9, '');  
-  end 
+class TicTacToe
+
+  def initialize(io: )
+    @board = Array.new(9, ' ')
+    @currentToken = 'X'
+    @io = io
+  end
+
+  def render_board
+    io.puts "#{@board[0]}|#{@board[1]}|#{@board[2]}"
+    io.puts "------"
+    io.puts "#{@board[3]}|#{@board[4]}|#{@board[5]}"
+    io.puts "------"
+    io.puts "#{@board[6]}|#{@board[7]}|#{@board[8]}"
+    board
+  end
 
   def newGame
-    
-  end 
-end 
+   @board = Array.new(9, ' ')
+  end
+
+ def placeToken(position)
+    board[position] = currentToken
+    render_board
+ end
+
+ def currentPlayer
+    @currentToken
+ end
+
+ def player_turn
+   desiredPosition = io.gets.to_i
+   io.puts "Select a position to place your token [1 - 9]: "
+
+   unless board[desiredPosition] == ' '
+      io.puts "That position is taken!"
+   else
+    placeToken( desiredPosition )
```

Comment:
Looks like the indentation is inconsistent between lines `36` and `38`. I also noticed you received comments on extraneous whitespace down below. I'd echo what @damonkelley said about using an autoformatter. I think `rubocop` is the standard linter used in Ruby: https://rubocop.readthedocs.io/en/stable/